DungeonGame.py

```diff
@@ -18,17 +18,20 @@
 die = ("die", "Die", "I die", "I Die", "i die", "i Die", "dead", "Dead")
 inventory = ("Inventory", "inventory", "I", "i")
 quit = ("Quit", "quit", "Q", "q", "Exit", "exit", "E", "e")
+game_state = {}
+
 
 class room(object):
 
     def enter(self):
+        pass
```

Comment:
Throw a `pass` into any function that's a placeholder until it's ready to be defined.